frontend/src/editor/types.ts

```diff
@@ -19,9 +19,9 @@ export type Document = {
 };
 
 // we fire this event when the user clicks on a word and we want the player to skip through it
-export const TEXT_CLICK_EVENT = 'textClick';
-export class TextClickEvent extends CustomEvent<{ text: Text }> {
-  constructor(text: Text) {
-    super(TEXT_CLICK_EVENT, { detail: { text } });
+export const SEEK_TO_EVENT = 'textClick';
+export class SeekToEvent extends CustomEvent<{ start?: number }> {
```

Comment:
It's a bit weird this is now named by what it should do, not by what has happened?

Nevertheless, if we keep this the string should be updated as well.